vnext/Microsoft.ReactNative/IReactDispatcher.cpp

```diff
@@ -5,6 +5,8 @@
 #include "IReactDispatcher.h"
 #include "ReactDispatcherHelper.g.cpp"
 
+#include <glog/logging.h>
```

Comment:
>glog [](http://example.com/codeflow?start=10&length=4)

I think it is a good approach. I just do not know if our code compiles with glog. If it does, then it is great and we must proceed with the change.

Reply:
Yeah, it looks totally stubbed in our repo at least. https://github.com/microsoft/react-native-windows/blob/main/vnext/stubs/glog/logging.h